packages/paste-website/src/pages/components/alert/index.mdx

```diff
@@ -56,8 +61,14 @@ export const pageQuery = graphql`
 
 <contentwrapper>
 
+<Box order={2} marginLeft="space140" minWidth="size20">
+
+<QuestionMenu data={props.data.mdx} />
```

Comment:
Question for ya: why did you opt to refactor all these pages over putting the QuestionMenu in the ToC component? Another possibility too: making a PageAside component that wraps all of this into a single  component.

Reply:
Dig that PageAside idea. I'll run with that.

Reply:
Refactored into a PageAside component. I left the TOC and QuestionMenu as shortcodes in the PasteMdxProvider in case we ever decided to use those separately in the future.